changelogs/unreleased/gh-11055-luajit-fixes.md

```diff
@@ -0,0 +1,10 @@
+## bugfix/luajit
+
+Backported patches from the vanilla LuaJIT trunk (gh-11055). The following
+issues were fixed as part of this activity:
+
+* Fixed `setfenv()` / `getfenv()` with the negative argument given.
+* Fixed register allocation for stores into sunk values (gh-10746).
+* Fixed a crash if using a Lua C function as a vmevent handler for trace
```

Comment:
```suggestion
* Fixed a crash when using a Lua C function as a vmevent handler for trace
```

Reply:
Negative inputs are still not allowed. This fixes the compiler (UBSan) warning.
Rephrasing as the following:
```
Fixed compiler warning in `setfenv()` / `getfenv()` with negative levels as the argument.
```